lib/vhdl-linter.ts

```diff
@@ -230,6 +260,11 @@ export class VhdlLinter {
     for (const obj of this.file.objectList) {
       if (obj instanceof OReference) {
         obj.elaborate();
+        for (const pkg of packages) {
+          if (obj.lexerToken.getLText() === pkg.lexerToken.getLText()) {
+            obj.definitions.push(pkg);
+          }
+        }
```

Comment:
this might set packages as definitions of any random reference which is wrong. See `test_pkg_reference.vhd`

Reply:
This is no only done for reads with selected name (prefix.suffix) etc. I am not sure this fully fixes the problem. But I think its the best we can do without types